spec/imsc-hrm.html

```diff
@@ -570,7 +579,7 @@ <h2>Paint Regions</h2>
 
       <p>where CLEAR(E<sub>n</sub>) = 0 if n == 0; and CLEAR(E<sub>n</sub>) = 1 otherwise.</p>
 
-      <p class='note'>To ensure consistency of the Presentation Buffer, a new <a>Intermediate Synchronic Document</a> requires
+      <p class='note'>To ensure consistency of the Front Buffer, a new <a>Intermediate Synchronic Document</a> requires
```

Comment:
Possibly a query for a separate issue, but I've seen real world issues where implementations actually cannot display the first ISD at its begin time if its begin time is too small, e.g. 0s. I wonder if we should actually include the initial CLEAR for _all_ ISDs including the first one...

Reply:
Addressed at https://github.com/w3c/imsc-hrm/pull/50/commits/5a9d4f5a28700613dccd3ea930c79b16d100dae6